src/gui/transferlistmodel.cpp

```diff
@@ -46,66 +46,42 @@
 
 namespace
 {
-    bool isDarkTheme()
-    {
-        const QPalette pal = QApplication::palette();
-        // QPalette::Base is used for the background of the Treeview
-        const QColor &color = pal.color(QPalette::Active, QPalette::Base);
-        return (color.lightness() < 127);
-    }
 
```

Comment:
In case you need more updates, this empty line should be removed too.

Reply:
Done.